packages/gunshi/src/types.ts

```
@@ -527,21 +518,30 @@ export interface TranslationAdapter<MessageResource = string> {
  * @param ctx A {@link CommandContext | command context}
  * @returns void or string (for CLI output)
  */
-export type CommandRunner<A extends Args = Args> = (
-  ctx: Readonly<CommandContext<A>>
+export type CommandRunner<A extends Args = Args, E extends Record<string, unknown> = {}> = (
+  ctx: Readonly<CommandContext<A, E>>
 ) => Awaitable<void | string>
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)